### PR TITLE
Fix base level psammead issue in talos

### LINKED ISSUE #1863: Talos fails to run
**Describe the bug**
Talos sometimes fails to run. This happens when manually invoked or via a publish

```
* Running "npx npm-check-updates @bbc/gel-foundations, @bbc/psammead-assets, @bbc/psammead-styles --packageFile package.json -u -a --jsonUpgraded && npx lerna exec --parallel --no-bail -- npx npm-check-updates @bbc/gel-foundations, @bbc/psammead-assets, @bbc/psammead-styles -u -a --jsonUpgraded"

TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received type undefined
```

https://ci.news.tools.bbc.co.uk/blue/organizations/jenkins/psammead/detail/latest/524/pipeline

**Expected behaviour**
It runs correctly

- ~[ ] This bug fix is expected to need manual testing.~

**Additional context**

### PR DESCRIPTION
**Pull request by dr3**
Resolves #1863

**Overall change:** Adds edge case to `getPackagePath` to allow getting base level package in jenkins

**Why:**

getPackages works on `new RegExp(`/${packageName}$`).test(packagePath)` where it expects the psammead folder to be `/psammead` or `pathtodir/psammead` this doesnt work on jenkins where the dir is `/var/lib/jenkins/workspace/psammead_debug-talos-Y2WYLDFQSDJ2QSYV4MDRV2AVRFVDIXQWB5T3V53L2PKQUIWNQJXA` 

Because of this ive added an exception and tests

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

File: scripts/utilities/getPackagePath/index.js [hunk 1]
```diff
@@ -4,5 +4,13 @@ module.exports = packageName => {
   const matchesPath = packagePath =>
     new RegExp(`/${packageName}$`).test(packagePath);
 
-  return getPackages().find(matchesPath);
+  const match = getPackages().find(matchesPath);
+
+  // covers case in jenkins where the base level `psammead` folder isnt
+  // simply called `psammead` but instead has a hash on the end
+  if(!match && packageName == 'psammead') {
```
Comment (thekp) [inline scripts/utilities/getPackagePath/index.js]:
```suggestion
  if (!match && packageName === 'psammead') {
```

CI complaining about

File: scripts/utilities/getPackagePath/index.js [hunk 2]
```diff
@@ -4,5 +4,13 @@ module.exports = packageName => {
...
+    return getPackages().find(packagePath => !packagePath.includes('/packages/'));
```
Comment (thekp) [inline scripts/utilities/getPackagePath/index.js]:
```suggestion
    return getPackages().find(
      packagePath => !packagePath.includes('/packages/'),
    );
```

File: scripts/utilities/getPackagePath/index.test.js [hunk 3]
```diff
@@ -1,20 +1,50 @@
...
-    expect(packagePath1).toEqual(
-      '/psammead/packages/components/psammead-brand',
-    );
-    expect(packagePath2).toEqual(
-      '/psammead/packages/components/psammead-image',
-    );
+    it('should return the path of the package', () => {
+      expect(getPackagePath('psammead')).toEqual(
+        '/psammead',
+      );
```
Comment (thekp) [inline scripts/utilities/getPackagePath/index.test.js]: prettier is saying this should be single line

**Review by thekp — CHANGES_REQUESTED** [review]
a few formatting problems on CI

File: scripts/utilities/getPackagePath/index.js (hunk 2 above)
Comment (dr3) [reply scripts/utilities/getPackagePath/index.js]: Done :)

File: scripts/utilities/getPackagePath/index.test.js (hunk 3 above)
Comment (dr3) [reply scripts/utilities/getPackagePath/index.test.js]: Done :)

File: scripts/utilities/getPackagePath/index.js (hunk 1 above)
Comment (dr3) [reply scripts/utilities/getPackagePath/index.js]: Done :)

**Review by thekp — APPROVED** [review]
nice 👍

**Review by jamesbhobbs — APPROVED** [review]
